### PR TITLE
docs: styling tweaks

### PR DESCRIPTION
**Pull request by layershifter**
## Color palette update

image

⬇️⬇️⬇️

image

## Fonts

Segoe UI fonts are used for text.

## Prism theme 

Now uses VSCode themes.

image

⬇️⬇️⬇️

image

## "code output" divider

image

⬇️⬇️⬇️

image

## Theme controls/Github link

image

⬇️⬇️⬇️

image

### CODE REVIEW

File: apps/website/src/components/OutputTitle/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styles from './styles.module.css';
```
Comment (layershifter) [inline apps/website/src/components/OutputTitle/index.js:2]: These should `makeStyles()` call, but Docosaurus doesn't support CSS-in-JS hooks for SSR setup: https://github.com/facebook/docusaurus/issues/3236

File: apps/website/src/theme/Toggle/index.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      <GithubIcon />
+    </a>
+  );
+}
+
+export default function () {
+  return (
+    <div className={styles.container}>
+      <ThemeToggle />
+      <GithubLink />
```
Comment (layershifter) [inline apps/website/src/theme/Toggle/index.js:94]: `GithubLink` should be just an item in navbar, but there is no way other way to React components to Navbar currently...

File: apps/website/package.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "name": "@griffel/website",
```
Comment (layershifter) [inline apps/website/package.json:2]: Sometimes docusaurus checks for updates and if `package.json` it fails any CLI command.

Comment (ling1726) [reply apps/website/package.json:2]: not sure I understood your comment

File: apps/website/src/theme/Toggle/index.js (hunk 2 above)
Comment (ling1726) [reply apps/website/src/theme/Toggle/index.js:94]: Maybe rename the folder ? this isn't a `Toggle` component anymore is it ?

File: apps/website/package.json (hunk 3 above)
Comment (layershifter) [reply apps/website/package.json:2]: Yeah, missed a few words there, sorry. 

- Docusaurus has update notifier
- It reads `package.json` inside project

https://github.com/facebook/docusaurus/blob/770418f8d262c56cd3de5ba1087f972ab20da617/packages/docusaurus/bin/beforeCli.js#L84-L88

- If it's absent, it throws on any CLI command (`docusaurus build`, `docusaurus start`, etc.)

File: apps/website/src/theme/Toggle/index.js (hunk 2 above)
Comment (layershifter) [reply apps/website/src/theme/Toggle/index.js:94]: As I understood Docusaurus checks for folder existence and then replaces a component (_as this folder is registered nowhere_). If I will rename it, I will get default controls back.

Comment (ling1726) [reply apps/website/src/theme/Toggle/index.js:94]: ah ok, now I understand

Comment (ling1726) [reply apps/website/src/theme/Toggle/index.js:94]: Let's throw a comment there, so this does not get lost. Is there a docusaurus issue about the navbar ?

File: apps/website/src/theme/Toggle/index.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { useColorMode } from '@docusaurus/theme-common';
+import useIsBrowser from '@docusaurus/useIsBrowser';
+import styles from './styles.module.css';
+
+const SunIcon = () => (
```
[the remark this answers is not in the document]
Comment (layershifter) [reply apps/website/src/theme/Toggle/index.js]: Yup, moved them out

File: apps/website/src/theme/Toggle/index.js (hunk 2 above)
Comment (layershifter) [reply apps/website/src/theme/Toggle/index.js:58]: Added a comment. I failed to find an issue/feature request to link it 😿

File: apps/website/package.json (hunk 3 above)
Comment (ling1726) [reply apps/website/package.json:2]: small nit: you could leverage the `description` property to doc that requirement

Comment (layershifter) [reply apps/website/package.json:2]: Hmm, good idea, pushed

**Review by ling1726 — APPROVED** [review]

**Pull request merged**